src/test/java/Homework19.java

```diff
@@ -7,12 +7,11 @@ public class Homework19 extends BaseTest {
 public void deletePlaylist () {
 
     login("kbetestack@gmail.com", "te$t$tudent");
-    WebElement getPlaylist = driver.findElement(By.cssSelector("a[href*='22390']"));
+    WebElement getPlaylist = driver.findElement(By.cssSelector("a[href*='22380']"));
     getPlaylist.click();
-    WebElement delPlaylist = driver.findElement(By.cssSelector(".del btn-delete-playlist"));
+    WebElement delPlaylist = driver.findElement(By.cssSelector(".btn-delete-playlist"));
     delPlaylist.click();
 
```

Comment:
Please finish the homework 19!

**Please, validate that the playlist is deleted.**

1. Create a new file Homework19.java with @Test annotation in IntelliJ IDEA.

2. Create a test case deletePlaylist() using @Test annotations.

3. Use the helper/reusable methods we created.

4. Use @Parameters for passing baseUrl from the TestNG config file to the tests.

5. Navigate to "https://bbb.testpro.io/".

6. Log in with your credentials.

7. Choose a playlist.

8. Delete the playlist

  **9. Validate that the playlist is deleted.**
 
10. Create a new branch and commit your changes.

11. Push your code to a remote repository.

12. Create a pull request.